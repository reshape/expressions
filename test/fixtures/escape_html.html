{{ 'x&x' }}
{{ 'x"x' }}
x{{ lt }}x
x{{ gt }}x
