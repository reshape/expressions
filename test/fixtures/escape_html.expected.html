x&amp;x
x&quot;x
x&lt;x
x&gt;x
